server/routes/todo.js

```diff
@@ -26,10 +28,24 @@ router.put('/', function(req, res, next) {
 
 
 /* DELETE todo list entry*/
-router.delete('/', function(req, res, next) {
-    // TODO implementation deleting entry from our database
-
-    res.send('delete todo list entry'); // TODO change response after
+router.delete('/userId/:userId/entryId/:entryId', function(req, res, next) {
+    var originData, updatedArray = [];
```

Comment:
You need to add the authentication between here

Reply:
Actually, I am not sure about the meaning of authentication you mention. And I suggest not to put authentication in such a specific endpoint.